e2e/specs/patientSearch.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { test } from '../core';
+import { HomePage } from '../pages/homePage';
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, Patient } from '../commands';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('should be able to search patients by identifier', async ({ page, api }) => {
+  // extract details from the created patient
+  const openmrsIdentifier = patient.identifiers[0].display.split('=')[1].trim();
+  const firstName = patient.person.display.split(' ')[0];
+  const lastName = patient.person.display.split(' ')[1];
+
+  const homePage = new HomePage(page);
+  await homePage.goto();
+
+  await homePage.searchPatient(openmrsIdentifier);
+
+  await expect(homePage.floatingSearchResultsContainer()).toHaveText(/1 search results/);
+  await expect(homePage.floatingSearchResultsContainer()).toHaveText(new RegExp(firstName));
+  await expect(homePage.floatingSearchResultsContainer()).toHaveText(new RegExp(lastName));
```

Comment:
I'm not sure I see the utility of regexes instead of strings here?

Reply:
"toHaveText" expects the provided string to be fully contained within the element. If we need to check if the element contains the provided string, we must use regular expressions.

https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-text